recipe/build.sh

```diff
@@ -5,11 +5,14 @@ BUILD_DIR="${SRC_DIR}/builddir"
 
 # configure
 meson setup \
-	${BUILD_DIR} \
-	${SRC_DIR} \
-	--prefix ${PREFIX} \
-	--libdir "lib" \
-	-Ddebug=false
+    ${MESON_ARGS} \
+    --prefix ${PREFIX} \
+    --libdir "lib" \
+    -Ddebug=false \
+    ${BUILD_DIR} ${SRC_DIR}
 
-# build & install
+# build
+meson compile -C ${BUILD_DIR} -j ${CPU_COUNT}
```

Comment:
As you already mentioned the `CPU_COUNT` is not necessary, as well as the whole `compile` call since it is called anyways by `install`. It also doesn't hurt I guess.

Reply:
> but this isn't so straight forward to do

I had the same idea.
The main problem that I found that autotests are not easily reproducible outside of current `develop`.
Did you encounter additional problems?

Reply:
Btw, there are no plans to support `meson test` any time soon.
You would still have to test, as we do it in our CI: https://github.com/MODFLOW-USGS/modflow6/blob/develop/.github/workflows/ci-tests-gfortran-latest.yml

Reply:
One simple test will do to make sure it can read/write files, and give a zero exit status if successful. I'm running the classic [TWRI example](https://modflow6-examples.readthedocs.io/en/master/_examples/ex-gwf-twri.html) which doesn't have any problems under Linux. Windows can be addressed in a separate PR.

Reply:
That would be good first step.
Thanks!